Src/library/base/elks/kernel/file.e

```diff
@@ -171,6 +171,40 @@ feature -- Initialization
 			open_append: is_open_append
 		end
 
+	make_temporary_open
+			-- Create a unique temporary file name.
```

Comment:
why not "Create Current file in read/write mode with unique temporary file name." ?
or something similar.

Reply:
The file is open and the mode is read_write. Am I missing something?

Reply:
oh you mean the description of the feature. yes need to be updated.

Reply:
What about this?
`Create a file object with a unique temporary file name with read/write mode'